src/components/BulkEnrollmentPage/BulkEnrollement.scss

```diff
@@ -0,0 +1,4 @@
+.bg-brand-primary {
```

Comment:
Can we rename for correct spelling?

Reply:
I can't spell enrollment or license to save my life. Will fix.